torch/_dynamo/output_graph.py

```diff
@@ -1893,6 +1839,8 @@ def __init__(
         # backward recomputation of the checkpoint region doesn't affect its correctness.
         self.allow_side_effects_under_checkpoint = False
 
+        self.level: int = parent.level + 1 if parent is not None else 0
```

Comment:
level just for debugging? call it "self.debug_level" or something so that people don't get confused when there are two subtracers with the same level